packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTypeaheadSelectList/CWTypeaheadSelectList.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Select, { components } from 'react-select';
+import { ComponentType } from '../../types';
+import './CWTypeaheadSelectList.scss';
+import { CWIcon } from '../../cw_icons/cw_icon';
+
+type SelectListOption = {
+  value: any;
```

Comment:
i think we can make it a bit specific? like string | number | boolean | object etc?